proto/mixer.proto

```diff
@@ -526,6 +526,16 @@ message RelatedPlacesInfo {
   reserved 4, 5, 6;
 }
 
+message GetStatVarPathRequest {
+  // DCID of the stat var (group)
+  string id = 1;
+}
+
+message GetStatVarPathResponse {
+  // A list of DCIDs from the queried stat var to the root of stat var hierarchy.
```

Comment:
Clarify perhaps that it is a single path, out of possibly multiple, to the root. And clarify the order?

Reply:
Done